dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/AbstractJdbcTask.java

```diff
@@ -61,8 +65,15 @@ public abstract class AbstractJdbcTask<T> extends AbstractTask<T> {
 
   @CheckForNull private Class<? extends Enum<?>> headerClass;
 
+  private final boolean isAssessment;
+
   public AbstractJdbcTask(@Nonnull String targetPath) {
+    this(targetPath, false);
+  }
+
+  public AbstractJdbcTask(@Nonnull String targetPath, boolean isAssessment) {
```

Comment:
I think this is actually a case for subclassing (doesn't have to be done in this PR). A factory that has knowledge about parameters could control which class is actually created.

Reply:
Adding this to the `TODO` list